azerothcore/azerothcore-wotlk
# fix(DB/creature): Move Megelon

## Changes Proposed:
-  Move Megelon

## Issues Addressed:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/4512 ~~(at least partially, the rest is disputed for now!)~~
- https://youtu.be/T_N08d9wvzY?t=148 (from a later expansion, I tried to copy this position)
- https://www.wowhead.com/npc=16475/megelon#comments comment says he moved in 3.3.5



## Tests Performed:
- Applied SQL, started a new Draenei character



## How to Test the Changes:

- start a Draenei character
- Notice location of Megelon
- Altenatively, do: `.go c id 16475`

## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
